docs/docs/manual/starting.md

```diff
@@ -80,24 +76,22 @@ This can be useful if you want to pre-select a specific number of rows from your
 
 This can also be useful if you would like to paste in a list of URLs, which you can use later to fetch the data online and build columns with. 
 
-
 #### Database (SQL)
 
 You may want to pull the latest dataset directly from an online database. This could include an online catalogue, a Wordpress or similar content management system, or a [digital repository or collection management system](https://bits.ashleyblewer.com/blog/2017/08/09/collection-management-system-collection/).
 
 OpenRefine can connect to PostgreSQL, MySQL, MariaDB, and SQLite database systems. It will automatically populate the “Port” field based on which of these you choose, but you can manually edit this if needed.
 
-To import data directly from a database online, you will need to do two things:
-
+If you have a downloaded database (`.db`) file, you can supply the path to the file on your computer directly in the “Database” field at the bottom of the form. You can leave the rest of the fields blank.
 
+To import data directly from a database online, you will need to do two things:
 
 *   Add OpenRefine (running from your computer) to an account authorized to access your database
 *   Set up OpenRefine to access that database using that account 
 
 ![A screenshot of connecting to a database](img/databaseconnect.jpg "A screenshot of connecting to a database.")
 
-
-Log in to your hosting provider. Get the database type (such as MySQL), database name, and the URL (either an IP address, such as `127.0.0.1`, or the domain that uses the database, such as _https://openrefine.org/category/blog.html_). Then look at the accounts authorized for access. You may wish to create a new account just for OpenRefine, or add OpenRefine to an existing account. 
+Log in to your hosting provider. Get the database type (such as MySQL), database name, and the URL (either an IP address, such as `127.0.0.1`, or the domain that uses the database). Then look at the accounts authorized for access. You may wish to create a new account just for OpenRefine, or add OpenRefine to an existing account. 
```

Comment:
I did try the list in one draft but found the details uneven. This section is otherwise fixed for the time being, according to the Mayer Stephens Detente of July 2020. :)